config/_default/menus/menus.en.yaml

```diff
@@ -534,6 +534,16 @@ main:
     parent: containers
     identifier: ecs_fargate
     weight: 6
+  - name: Troubleshooting
+    url: containers/troubleshooting/
```

Comment:
With no target file here, the nav does weird things. I think you should probably create a `/containers/troubleshooting/_index.md` file with a `{{< whatsnext >}}` list of (one) links to the child troubleshooting pages?